modules/admin_manual/pages/enterprise/security/oidc/index.adoc

```diff
@@ -0,0 +1,109 @@
+= OpenID Connect (OIDC)
+:toc: right
+:toclevel: 2
+:panva-node-oidc-provider-url: https://github.com/panva/node-oidc-provider
+:openid-connect-frontchannel-logout-url: https://openid.net/specs/openid-connect-frontchannel-1_0.html
+:openid-connect-url: https://openid.net/connect/
+:openid-config-url: http://localhost:3000/.well-known/openid-configuration
+
+== Introduction
+
+"OpenID Connect 1.0 is a simple identity layer on top of the OAuth 2.0 protocol. It allows Clients to verify the identity of the End-User based on the authentication performed by an Authorization Server, as well as to obtain basic profile information about the End-User in an interoperable and REST-like manner.
+OpenID Connect allows clients of all types, including Web-based, mobile, and JavaScript clients, to request and receive information about authenticated sessions and end-users. The specification suite is extensible, allowing participants to use optional features such as encryption of identity data, discovery of OpenID Providers, and session management, when it makes sense for them."
+-- {openid-connect-url}, "What is OpenID Connect?"
+
+=== Configure ownCloud
+
+Please refer to xref:configuration/server/config_sample_php_parameters.adoc#oidc-configuration[the OIDC configuration example] in the sample configuration documentation for how to configure ownCloud to work with OIDC.
+
+=== Setup Within the OpenID Provider
+
+When registering ownCloud as OpenID Client use `https://cloud.example.net/index.php/apps/openidconnect/redirect` as the redirect URL.
+In case {openid-connect-frontchannel-logout-url}[OpenID Connect Front-Channel Logout 1.0] is supported, please enter `https://cloud.example.net/index.php/apps/openidconnect/logout` as the logout URL within the client registration of the OpenID Provider.
+We require `frontchannel_logout_session_required` to be true.
+
+=== Setup Service Discovery
+
+To allow other clients to use OpenID Connect when talking to ownCloud, please set up a redirect on the webserver to point `.well-known/openid-configuration` to `/index.php/apps/openidconnect/config`.
+Here is an .htaccess example
+
+[source]
+----
+RewriteRule ^\.well-known/openid-configuration /index.php/apps/openidconnect/config [R=301,L]
+----
+
+TIP: Service discovery is not mandatory at the moment as no client, currently, supports it.
```

Comment:
Just wrong

Reply:
Odd, as https://github.com/owncloud/openidconnect#setup-service-discovery says say.

Reply:
Well - development on the clients did evolve as well in the past months since we started to document this. :man_shrugging:

Reply:
Might be worth updating that README.